### DISCUSSION TITLE
Thank you Mantine!

### DISCUSSION THREAD

**Discussion by OmkoBass** [mantinedev/mantine#259] [Kind words]
Out of all react component libraries that I have ever seen this one is the most straight forward, easy to use, well documented and really beautiful. I plan on switching and using this full time. Just wanted to say huge thanks to the people that made this.

**Comment by rtivital** [comment]
Hi Omer, thanks for kind words! Feel free to reach out on Discord if you have any questions

**Comment by amok** [comment]
I am an MUI user. Today I discovered Mantine and liked it at first sight. The only thing which worries me is the lib's future and its maintenance in the future. Looking at MUI - it has matured into a (hopefully) profitable company with paid employees, it is very popular (Google's Material UI brand helped) and there is almost no doubt it'll keep evolving. On the other hand, we see a lot of OS libs getting abandoned, or stop evolving. @rtivital what are your plans on Mantine?

**Reply by rtivital** [reply]
I'm not planning to create a company around a React library like MUI. My goals with this project are:
- build components and hooks that I can use for work and personal projects without putting too much effort into overriding default styles
- have fun in the process: meet cool people, learn new things, etc.

Everything went pretty smooth so far.

**Reply by KlaasMoukangwe** [reply]
Really Great project. I am going to use it for my company site.

Will give you feedback

**Comment by Developer-Mominov** [comment]
Mantine is a very good UI library, thanks to its creators.

**Comment by zainulhassan815** [comment]
Mantine is pretty awesome. Its really simple and straight forward. I would like the team to have youtube channel for general tutorials and library updates.

**Comment by seaweasel** [comment]
This looks promising. I haven't dived in yet, but it looks like it will be easy to shim Mantine components in along existing libraries like MUI for a real head-to-head comparison.
